<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="3-AgregarLogistica_20_TestSuites" time="7.592" tests="1" failures="0" errors="1">
   <testsuite name="3-AgregarLogistica_20_TestSuites" tests="1" failures="0" errors="1" time="7.592" timestamp="2020-01-28 16:03:14" hostname="kaizen - DESKTOP-38N4VV4" id="Test Suites/Flujo_funcionalidades_Pedidos_20pies/3-AgregarLogistica_20_TestSuites">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\kaizen\\git\\AutomatizacionKatalon\\Reports\\20200128_160312\\Flujo_funcionalidades_Pedidos_20pies\\3-AgregarLogistica_20_TestSuites\\20200128_160312"/>
         <property name="logFiles" value="C:\\Users\\kaizen\\git\\AutomatizacionKatalon\\Reports\\20200128_160312\\Flujo_funcionalidades_Pedidos_20pies\\3-AgregarLogistica_20_TestSuites\\20200128_160312\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="kaizen - DESKTOP-38N4VV4"/>
         <property name="hostAddress" value="192.168.1.114"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="7.2.1.1"/>
      </properties>
      <testcase name="Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies" time="7.174" classname="Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies" status="ERROR">
         <error type="ERROR" message="Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'tc_variable' for test data 'Cargos_Logisticos20pies.xlsx'. Possible values are '{Idioma,Usuario,Contrasena,IdPedido,Id_Cliente,Estatus_Pedido,Creado_desde,IDCargo,NombreCargo,IDCargo2,NombreCargo2}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at com.kms.katalon.core.testdata.TestData$getValue.call(Unknown Source)
	at TC_Generar_CargosLogisticos_20pies.run(TC_Generar_CargosLogisticos_20pies:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1580241792118.run(TempTestSuite1580241792118.groovy:36)
"/>
         <system-out><![CDATA[2020-01-28 16:03:14 - [TEST_CASE][ERROR] - Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies: Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'tc_variable' for test data 'Cargos_Logisticos20pies.xlsx'. Possible values are '{Idioma,Usuario,Contrasena,IdPedido,Id_Cliente,Estatus_Pedido,Creado_desde,IDCargo,NombreCargo,IDCargo2,NombreCargo2}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at com.kms.katalon.core.testdata.TestData$getValue.call(Unknown Source)
	at TC_Generar_CargosLogisticos_20pies.run(TC_Generar_CargosLogisticos_20pies:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1580241792118.run(TempTestSuite1580241792118.groovy:36)


2020-01-28 16:03:15 - [TEST_STEP][PASSED] - python.Ex_logistica_pedidos_20pies.runPython(): python.Ex_logistica_pedidos_20pies.runPython is PASSED

2020-01-28 16:03:21 - [MESSAGE][PASSED] - python.Ex_logistica_pedidos_20pies.runPython is PASSED

2020-01-28 16:03:21 - [TEST_STEP][PASSED] - excel = getExcelDataWithDefaultSheet("Cargos_Logisticos20pies.xlsx", "Sheet1", true): null

2020-01-28 16:03:22 - [TEST_STEP][PASSED] - excel.getValue("tc_variable", 1): null]]></system-out>
         <system-err><![CDATA[2020-01-28 16:03:14 - [TEST_CASE][ERROR] - Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies: Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'tc_variable' for test data 'Cargos_Logisticos20pies.xlsx'. Possible values are '{Idioma,Usuario,Contrasena,IdPedido,Id_Cliente,Estatus_Pedido,Creado_desde,IDCargo,NombreCargo,IDCargo2,NombreCargo2}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at com.kms.katalon.core.testdata.TestData$getValue.call(Unknown Source)
	at TC_Generar_CargosLogisticos_20pies.run(TC_Generar_CargosLogisticos_20pies:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1580241792118.run(TempTestSuite1580241792118.groovy:36)]]></system-err>
      </testcase>
      <system-out><![CDATA[2020-01-28 16:03:14 - [TEST_SUITE][ERROR] - 3-AgregarLogistica_20_TestSuites: Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'tc_variable' for test data 'Cargos_Logisticos20pies.xlsx'. Possible values are '{Idioma,Usuario,Contrasena,IdPedido,Id_Cliente,Estatus_Pedido,Creado_desde,IDCargo,NombreCargo,IDCargo2,NombreCargo2}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at com.kms.katalon.core.testdata.TestData$getValue.call(Unknown Source)
	at TC_Generar_CargosLogisticos_20pies.run(TC_Generar_CargosLogisticos_20pies:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1580241792118.run(TempTestSuite1580241792118.groovy:36)]]></system-out>
      <system-err><![CDATA[2020-01-28 16:03:14 - [TEST_SUITE][ERROR] - 3-AgregarLogistica_20_TestSuites: Test Cases/Secuencias de Ejecucion/TC_Generar_CargosLogisticos_20pies FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'tc_variable' for test data 'Cargos_Logisticos20pies.xlsx'. Possible values are '{Idioma,Usuario,Contrasena,IdPedido,Id_Cliente,Estatus_Pedido,Creado_desde,IDCargo,NombreCargo,IDCargo2,NombreCargo2}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at com.kms.katalon.core.testdata.TestData$getValue.call(Unknown Source)
	at TC_Generar_CargosLogisticos_20pies.run(TC_Generar_CargosLogisticos_20pies:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1580241792118.run(TempTestSuite1580241792118.groovy:36)]]></system-err>
   </testsuite>
</testsuites>
